lib/govuk_index/presenters/elasticsearch_presenter.rb

```diff
@@ -64,7 +64,13 @@ def common_fields
     end
 
     def details
-      @_details ||= DetailsPresenter.new(payload["details"])
+      @_details ||= begin
+        DetailsPresenter.new(
+          details: payload["details"],
+          format: common_fields.format,
+          sanitiser: IndexableContentSanitiser.new
+        )
+      end
```

Comment:
why is this wrapped in a begin/end block

Reply:
multi line memoization